config/routes.rb

```diff
@@ -678,8 +678,10 @@
 
     scope path: '/applications/:application_form_id' do
       get '/' => 'application_forms#show', as: :application_form
+      get '/add-course' => 'add_course#new', as: :add_course_for_candidate
```

Comment:
For least surprise, how about `GET courses/new` and `POST courses/`?

Reply:
Yep these routes need to be changed as you described 👍

Reply:
Lovely, thank you!